tests/FrameMapper_Tests.cpp

```diff
@@ -31,33 +31,35 @@
 #include "UnitTest++.h"
 // Prevent name clashes with juce::UnitTest
 #define DONT_SET_USING_JUCE_NAMESPACE 1
-#include "OpenShot.h"
+#include "CacheMemory.h"
+#include "Clip.h"
+#include "DummyReader.h"
+#include "FFmpegReader.h"
+#include "Fraction.h"
+#include "Frame.h"
+#include "FrameMapper.h"
+#include "Timeline.h"
 
 using namespace std;
 using namespace openshot;
 
-TEST(FrameMapper_Get_Valid_Frame)
+SUITE(FrameMapper) {
+TEST(NoOp_GetMappedFrame)
 {
 	// Create a reader
 	DummyReader r(Fraction(24,1), 720, 480, 22000, 2, 5.0);
 
-	// Create mapping between 24 fps and 29.97 fps using classic pulldown
-	FrameMapper mapping(&r, Fraction(30000, 1001), PULLDOWN_CLASSIC, 22000, 2, LAYOUT_STEREO);
+	// Create mapping between 24 fps and 24 fps without pulldown
+	FrameMapper mapping(&r, Fraction(24, 1), PULLDOWN_NONE, 22000, 2, LAYOUT_STEREO);
+	CHECK_EQUAL("FrameMapper", mapping.Name());
 
-	try
-	{
-		// Should find this frame
-		MappedFrame f = mapping.GetMappedFrame(125);
-		CHECK(true); // success
-	}
-	catch (OutOfBoundsFrame &e)
-	{
-		// Unexpected failure to find frame
-		CHECK(false);
-	}
+	// Should find this frame
+	MappedFrame f = mapping.GetMappedFrame(125);
+	CHECK_EQUAL(125, f.Odd.Frame);
+	CHECK_EQUAL(125, f.Even.Frame);
```

Comment:
I'm an idiot. The problem is that at 24fps, there are only 120 frames in the reader.

```suggestion
	MappedFrame f = mapping.GetMappedFrame(100);
	CHECK_EQUAL(100, f.Odd.Frame);
	CHECK_EQUAL(100, f.Even.Frame);
```